modules/library/data/src/test/java/org/geotools/data/memory/MemoryDataStoreBoundsTest.java

```diff
@@ -53,8 +56,9 @@ protected void setUp() throws Exception {
         data = new MemoryDataStore();
         data.addFeatures(roadFeatures);
         
-        SimpleFeatureType riverTypeWithCrs = SimpleFeatureTypeBuilder.retype(riverType, DefaultGeographicCRS.WGS84);
+        SimpleFeatureType riverTypeWithCrs = SimpleFeatureTypeBuilder.retype(riverType, CRS.decode("EPSG:4326"));
```

Comment:
DefaultGeographicCRS.WGS84 and EPSG:326 represent the same idea mathematically. We *may* get a slight difference based on axis order settings.

I got no problem using EPSG:4326 here as it is consistent with testing reprojection code.